### PR TITLE
feat(root): final visual and program-completion convergence

### PR DESCRIPTION
**Pull request by Aptymok**
## Trigger
Founder/ROOT directed immediate completion of the remaining FULL REMAKE internal development, including the already-selected ROOT visual system and a completion controller that can actually prove `SATISFIED`.

## SFI PRECHECK
Owner: existing `/root` authority/read surface + SFI-00 Program Completion Controller; WS-08 independent assurance.
Existing capability inspected: canonical ROOT authority/read contracts, founder console read model, canonical surface architecture, `scripts/sfi-program-completion.mjs`, #389/#405 completion governance, existing SFI Verify/release gates, PR #442 Vercel cost gate.
Absorb vs create decision: ABSORB. No new ROOT backend, writer, database authority, workstream or application. Visual change is a projection/lens over existing read contracts; completion proof extends the existing controller.
Authoritative writer: unchanged existing ROOT/governance/persistence owners. Completion receipts are evidence for controller classification, not a new institutional mutation plane.
Persistence/lineage impact: any completion receipt must bind requirement identity/text to non-empty durable evidence; stale/mismatched receipts fail closed. No simulated ROOT values may be persisted or presented as observed.
Front delta: absorb elected void/charcoal + restrained gold/amber control-room visual language into canonical ROOT, with explicit MISSING/DEGRADED/epistemic/provenance states.
Back delta: make evidence-backed `SATISFIED` reachable in the existing completion controller.
DB delta: NONE unless an already-existing canonical owner requires it; no new DB owner is authorized by this PR.
Redundancy removed: mockup-only/parallel ROOT interpretation and structurally unreachable SATISFIED state.
Execution/ROOT boundary: ROOT remains authority + institutional work queue; routine transitions remain automated under governed contracts. No authority expansion or automatic canon.
Rollback: revert this PR; no evidence/history deletion.
Verification: deterministic completion-receipt tests + ROOT visual/semantic QA + canonical architecture + typecheck/build + SFI Verify + Program Completion Controller + Main-Only convergence after active branches are integrated/disposed.

## Cost boundary
Automatic Vercel Git deployments are disabled by merged PR #442. Do not use production deployment for iteration. One final release only after exact canonical main + assurance when production observation is required for RETURN.

## Authority source
See `docs/program/decisions/D-2026-09-08-FINAL-ROOT-CONVERGENCE.md`.

### CODE REVIEW

**Comment by Aptymok** [comment]
@codex implement the accepted directive in this PR.

Implement both coupled repairs from `D-2026-09-08-FINAL-ROOT-CONVERGENCE.md` against current branch state:

1) Program Completion: make `SATISFIED` reachable only through durable requirement-bound evidence/receipts; add deterministic falsification tests for valid, stale/mismatched, evidence-empty and external-only cases. Do not reclassify code presence as completion.

2) ROOT visual: locate the current canonical `/root` UI/read model and absorb the selected SFI control-room visual system (void/charcoal, restrained gold/amber, red only for critical states, technical topology/network field, ten-module institutional composition) into that existing surface. Reuse current read contracts and writers. No simulated/hard-coded health metrics, no new ROOT backend, no new writer, no authority expansion. Preserve explicit MISSING/DEGRADED, epistemic class and provenance. Map obsolete mockup labels onto current canonical tasks instead of resurrecting deleted routes.

Add focused QA and wire it into existing relevant workflows/build without creating a new workstream. Keep Vercel production untouched. After implementation, request independent review on the exact HEAD.

File: docs/program/decisions/D-2026-09-08-FINAL-ROOT-CONVERGENCE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Status: ACCEPTED IMPLEMENTATION DIRECTIVE
+Authority: founder/ROOT instruction in active SFI-00 control-room session on 2026-09-08.
+Base: `ee2c999d7c5f5883d2dcac2112a06639b94bdec7`.
+
+## Decision
+
+The FULL REMAKE final internal convergence has two coupled repairs and MUST reuse existing canonical owners:
+
+1. Repair `scripts/sfi-program-completion.mjs` so `SATISFIED` is reachable only from durable, requirement-bound completion evidence. Presence of code alone remains insufficient. Missing/partial/external states remain explicit. The controller must fail closed on stale, mismatched or evidence-free completion receipts.
```
[the remark this answers is not in the document]
Comment (Aptymok) [reply docs/program/decisions/D-2026-09-08-FINAL-ROOT-CONVERGENCE.md:11]: Absorbed on rebuilt exact HEAD `b24f7edf89badc47fa8b062eca2cb2573e909e78`: the existing controller now runs a separate evidence-bound reconciliation pass (`scripts/sfi-program-completion-reconcile.mjs`) using `SFI-PROGRAM-COMPLETION-RECEIPTS-1.0`. `qa-sfi-program-completion-receipts.mjs` proves valid current receipts can promote SATISFIED while stale hashes, empty evidence and unobserved external requirements fail closed; the workflow executes both QA and reconciliation after base classification.

File: docs/program/decisions/D-2026-09-08-FINAL-ROOT-CONVERGENCE.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+2. Absorb the founder-selected ROOT visual system into the existing canonical `/root` surface/read contracts. Do not create a second ROOT backend, writer, authority plane or application.
```
[the remark this answers is not in the document]
Comment (Aptymok) [reply docs/program/decisions/D-2026-09-08-FINAL-ROOT-CONVERGENCE.md:12]: Absorbed on rebuilt exact HEAD `b24f7edf89badc47fa8b062eca2cb2573e909e78`: canonical `/root` remains `SfiConsole -> SfiRootWorkspace` and now carries `SFI-ROOT-VISUAL-2.0`, exactly ten canonical module lenses, explicit MISSING/DEGRADED and epistemic boundaries, and the elected void/charcoal + restrained gold/amber styling. `qa-sfi-root-final-visual.mjs` is wired into Program Completion CI and asserts no simulated/hardcoded healthy state or new writer.

File: scripts/lib/programCompletionReceipts.mjs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  if (receipt.requirementHash !== expectedHash) {
+    return { state: 'INVALID', satisfied: false, receipt, error: 'REQUIREMENT_HASH_MISMATCH', expectedHash };
+  }
+  if (!Array.isArray(receipt.evidence) || receipt.evidence.length === 0 || receipt.evidence.some((value) => typeof value !== 'string' || !value.trim())) {
+    return { state: 'INVALID', satisfied: false, receipt, error: 'COMPLETION_EVIDENCE_REQUIRED', expectedHash };
+  }
+  if (typeof receipt.verifiedBy !== 'string' || !receipt.verifiedBy.trim()) {
+    return { state: 'INVALID', satisfied: false, receipt, error: 'VERIFIER_REQUIRED', expectedHash };
+  }
+  if (receipt.returnState !== 'RETURN_PASS') {
```
[the remark this answers is not in the document]
Comment (Aptymok) [reply scripts/lib/programCompletionReceipts.mjs]: Absorbed on exact HEAD `d95b09d603645b4fd02349c99a98e079db23e7a3`. Receipt contract is now `SFI-PROGRAM-COMPLETION-RECEIPTS-1.1`: SATISFIED requires exact current HEAD, verifier exactly `SFI-08`, structured evidence references, and an evidence resolver that observes each reference. Reconciliation resolves git commits, successful exact-head GitHub workflow runs, and digest-bound repository RETURN/production/external receipt files. Self-asserted strings, wrong head, wrong verifier, unresolved evidence, and missing resolver all fail closed; deterministic QA covers each falsification.

File: scripts/sfi-program-completion-reconcile.mjs [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+const report = JSON.parse(fs.readFileSync(artifactPath, 'utf8'));
+const ledger = loadCompletionReceiptLedger(ledgerPath);
+const canonicalStatuses = ['SATISFIED','IN_PROGRESS','PARTIAL','MISSING','EXTERNAL_ACTION','SUPERSEDED_BY_AUTHORIZED_DECISION'];
+const invalidReceipts = [];
+let promoted = 0;
+
+for (const requirement of report.requirements ?? []) {
+  const external = requirement.status === 'EXTERNAL_ACTION';
+  const assessment = evaluateCompletionReceipt(requirement, ledger, { external });
```
[the remark this answers is not in the document]
Comment (Aptymok) [reply scripts/sfi-program-completion-reconcile.mjs]: Absorbed on exact HEAD `d95b09d603645b4fd02349c99a98e079db23e7a3`. Reconciliation no longer infers externality from mutable classification status; it derives the external boundary directly from the canonical requirement source/text. Therefore a matching open issue cannot turn an external requirement into an internal one, and `externalObserved:true` remains mandatory for any external-only SATISFIED receipt.

File: src/components/sfi/SfiRootWorkspace.css [hunk 5]
```diff
@@ -1 +1 @@
...
...2em;color:rgba(224,184,91,.42)}.rootToast{position:sticky;top:8px;z-index:20;border:1px solid rgba(201,168,75,.35);background:#0c0d0b;padding:10px 12px;display:flex;justify-content:space-between;gap:12px}.rootToast.error{border-color:rgba(185,74,74,.7)}.rootToast span{font-size:10px}.rootToast button{background:none;border:0;color:inherit;cursor:pointer}@media(max-width:1180px){.rootObserveLinks{grid-template-columns:repeat(4,minmax(0,1fr))}}@media(max-width:1050px){.rootPulse{grid-template-columns:repeat(2,minmax(0,1fr));}.rootObserveLinks{grid-template-columns:repeat(2,minmax(0,1fr))}}@media(max-width:900px){.rootWorkspace{padding:16px}.rootHeader{grid-template-columns:1fr;align-items:start}.rootReadState{align-items:flex-start;border-left:0;border-top:1px solid var(--root-line);padding:14px 0 0}.rootDecisionLayout{grid-template-columns:1fr}.rootDecisionQueue{max-height:none}.rootRule{align-items:flex-start;flex-direction:column}.rootRule span{text-align:left}.rootDossierHero{flex-direction:column}}@media(max-width:580px){.rootPulse,.rootObserveLinks{grid-template-columns:1fr}.rootObserveLead{flex-direction:column}.rootHeader h1{font-size:40px}.rootWorkspace:before{display:none}}
```
[the remark this answers is not in the document]
Comment (Aptymok) [reply src/components/sfi/SfiRootWorkspace.css]: Absorbed on exact HEAD `d95b09d603645b4fd02349c99a98e079db23e7a3`. Pulse values now render `MISSING` until `/api/root/interactive` has produced a successful base read; read failures project `DEGRADED` rather than zero. Each card carries `data-epistemic-state={readState}` and CSS renders that state dynamically instead of stamping `OBSERVED`. ROOT visual QA now asserts these invariants.
